species/arabidopsis/build-docker-embeddable.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#image name
+IMAGE_NAME="knetminer-arabidopsis-dev"
+
+# current $SPECIES_DIR.
+SPECIES_DIR=`pwd | rev | cut -d '/' -f1 | rev`
+
+#copy .dockerignore-template and .gitignore settings.
+cd ../..
+cp .dockerignore-template .dockerignore
+cat .gitignore >> .dockerignore
+
+# add all species/ sub-folders to .dockerignore except KnetMiner/pom.xml, common/ and species/$SPECIES_DIR.
+cd species/
+ls -1 | grep -v $SPECIES_DIR | sed 's/^/species\//' | grep -v pom.xml | grep -v common >> .dockerignore
+cd ..
+
+#pass docker image build parameters
+docker image build \
+    	--build-arg tax_id=3702  \
+    	--build-arg species_name="Arabidopsis thaliana" \
+    	--build-arg species_link_name=araknet \
+    	--build-arg keywords="arabidopsis, a.thaliana, knetminer, quickstart, demo" \
+    	--build-arg description="Arabidopsis Knetminer" \
+    	--build-arg reference_genome=true \
+  --build-arg species_dir="species/$SPECIES_DIR" \
+  --build-arg oxl_file="AraKNET_v42.oxl" \
+  --build-arg knetminer_port=8080 \
+  --build-arg embeddable=true \
```

Comment:
This line is the only difference between this file and `build-docker-dev.sh`. Not sure if it worth it to create this new file at all.

Reply:
yes, you could leave it just as a (documented) parameter in the build-docker.sh that developer users can set before deploying, instead of adding a new file.

Reply:
> (documented)
Where should I document it? Shall I add the information to Wiki page or it's better to just leave a comment in code?

Reply:
just a code comment should suffice for now, thanks

Reply:
Added the comment